plugins/Ntuplizer.cc

```diff
@@ -724,7 +724,7 @@ Ntuplizer::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
      theW.addDaughter(MET);
      addP4.set(theW);
 
-     if (theW.mass()<100. && isControl){
+     if (theW.tmass()<100. && isControl){
```

Comment:
Fix transverse mass method

```suggestion
     if (theW.mt()<100. && isControl){
```